src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1763,6 +1792,14 @@ private void setItemToCacheForUser(final AuthenticationRequest request,
             mTokenCacheStore.setItem(CacheKey.createMultiResourceRefreshTokenKey(request, userId),
                     new TokenCacheItem(request, result, true));
         }
+        
+        if (!StringExtensions.IsNullOrBlank(result.getFamilyClientId()) && !StringExtensions.IsNullOrBlank(userId)) {
```

Comment:
For setting MRRT above, why wasn't client ID used in the key? Also, curious as to why is userId checked here but not above.